modules/nodes-scheduler-descheduler-create-policy.adoc

```diff
@@ -11,44 +11,78 @@ You can configure the descheduler to remove pods from nodes that violate rules d
 
 To create the needed policy:
 
-. Create a YAML file similar to the following:
+. Create a Custom Resouce Definition in a YAML file similar to the following:
```

Comment:
When we use the operator, we don't need to create configmap.

Reply:
@ravisantoshgudimetla The CRD is automatically created? What is the name of the file? 
Does the user just need to `oc edit` the file to make changes?

Reply:
@ravisantoshgudimetla ^^